modules/rtm/R/invert.custom.R

```diff
@@ -176,7 +194,6 @@ invert.custom <- function(observed, invert.options,
   }
 
   # Precalculate quantities for first inversion step
-  rp1 <- tau_0 + nspec * nwl/2
   rsd <- 0.5
```

Comment:
0.5 is the for the residual standard deviation. `rsd` is updated by the MCMC while sampling, but by a Gibbs sampling step rather than Metropolis proposal-rejection, so it always has a 100% acceptance rate.